### PR TITLE
[#134003195] Update Docs so it tells the site Integrator about new Pl…

### PR DESCRIPTION
**Pull request by pustovalov**
…acements

### CODE REVIEW

File: source/campaigns.rst [hunk 1]
```diff
@@ -25,13 +25,26 @@ To start editing Views simple visit ``Editor`` page from the Campaign dashboard.
...
-  <h2>Site Placements & Campaign Structure</h2>
+  <h2>Campaign Types and Placements</h2>
+
++-------------------------------+---------------------------------+
+| .. toctree::                  | .. toctree::                    |
+|   :maxdepth: 2                |   :maxdepth: 2                  |
+|                               |                                 |
+|   campaigns/campaign_types    |   campaigns/campaign_placements |
+|                               |                                 |
+|                               |                                 |
```
Comment (zhuravel) [inline source/campaigns.rst]: Why do you need empty lines here?

File: source/campaigns/campaign_placements/floating_widget.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+:ref:`Learn more about Placements <campaigns/campaign_placements>`
+
+Integration instructions
+------------------------
+
+Floating Widget Placement doesn’t require any additional integration.
+If you have already added Talkable to your main layout (completed General Integration
+for Custom platform, or installed Shopify or Magento extensions), then you are all done.
+|br|
+**Note:** For Magento site just make sure the Floating Widget Popup is enabled in the Talkable extension settings on your Magento Admin site.
```
Comment (zhuravel) [inline source/campaigns/campaign_placements/floating_widget.rst]: "on your Magento Admin site" → "in your Magento Admin Panel"

File: source/campaigns/campaign_placements/gleam.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+:ref:`Learn more about Placements <campaigns/campaign_placements>`
+
+Integration instructions
+------------------------
+
+Gleam Placement doesn’t require any additional integration.
+If you have already added Talkable to your main layout (completed General Integration
+for Custom platform, or installed Shopify or Magento extensions), then you are all done.
+|br|
+**Note:** For Magento site just make sure the Gleam is enabled in the Talkable extension settings on your Magento Admin site.
```
Comment (zhuravel) [inline source/campaigns/campaign_placements/gleam.rst]: Same here.

File: source/campaigns/campaign_types.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+from various high traffic locations on the website as well as outside the website.
+You can set up multiple Invite campaigns to run simultaneously.
+
+.. image:: /_static/img/campaign_types/invite.png
+   :alt: Campaign Type - Invite,
+   :class: is-minimal
+
+Advocate Dashboard
+------------------
+The Advocate Dashboard campaigns provide the Advocates a great insight on each of
```
Comment (zhuravel) [inline source/campaigns/campaign_types.rst]: Lets add empty lines after headers.

File: source/campaigns/campaign_placements/floating_widget.rst [hunk 5]
```diff
@@ -17,7 +17,7 @@ Floating Widget Placement doesn’t require any additional integration.
 If you have already added Talkable to your main layout (completed General Integration
 for Custom platform, or installed Shopify or Magento extensions), then you are all done.
 |br|
-**Note:** For Magento site just make sure the Floating Widget Popup is enabled in the Talkable extension settings on your Magento Admin site.
+**Note:** For Magento site just make sure the Floating Widget Popup is enabled in the Talkable extension settings in your Magento Admin site.
```
Comment (zhuravel) [inline source/campaigns/campaign_placements/floating_widget.rst]: "Magento Admin site" → "Magento Admin Panel" everywhere.

File: source/campaigns/site_placements/post_purchase.rst [hunk 6]
```diff
@@ -1,121 +0,0 @@
...
-.. image:: /_static/img/site_placements/pp.png
-   :alt: Talkable campaign flow — Post Purchase template,
-   :class: is-minimal
-
-Basic Integration
------------------
-
-When the Talkable JS integration library is all set (:ref:`see 1 step here <ecommerce/custom>`) on the order confirmation page the following integration script needs to be included:
-
-.. include:: /samples/ecommerce/custom/post_purchase_basic.rst
```
Comment (zhuravel) [inline source/campaigns/site_placements/post_purchase.rst]: @pustovalov Where can the user see post purchase integration JS example now?

**Comment by iurevych** [comment]
@zhuravel https://www.talkable.com/sites/bz2/integration#page_other_second_step

**Pull request merged**